fab/inventory/tsung

```diff
@@ -0,0 +1,36 @@
+[proxy]
+hqproxy4.internal.commcarehq.org
+
+[webworkers]
+hqdjango4-staging.internal.commcarehq.org
+hqdjango5-staging.internal.commcarehq.org
+
+[couchdb]
+hqdb0-staging.internal.commcarehq.org
+
+[postgresql]
+hqdb0-staging.internal.commcarehq.org
+
+[rabbitmq]
+hqdb0-staging.internal.commcarehq.org
+
+[zookeeper]
+hqkafka0-staging.internal.commcarehq.org
+
+[kafka]
+hqkafka0-staging.internal.commcarehq.org kafka_broker_id=843
+
+[celery]
+hqdb0-staging.internal.commcarehq.org
+
+[pillowtop]
+hqdb0-staging.internal.commcarehq.org
+
+[redis]
+hqdb0-staging.internal.commcarehq.org
+
+[elasticsearch]
+hqes1-staging.internal.commcarehq.org elasticsearch_node_name=hqes1-tsung
```

Comment:
I don't think we need ES at all since we're just going to be benchmarking form submissions